core/controllers/contributor_dashboard_test.py

```diff
@@ -558,9 +562,10 @@ def test_handler_with_no_target_language_code_raises_exception(self):
                 'state_name': 'End State',
                 'content_ids': '["content"]',
             }, expected_status_int=400)
+
+        error_msg = ('Missing key in handler args: target_language_code.')
```

Comment:
Please remove brackets for single line strings(ditto throughout the file).

Reply:
Done